### PR TITLE
test: Add Cloud Firewall to LKE nodes after integration test execution

### PR DESCRIPTION
**Pull request by ykim-1**
## 📝 Description

Currently there isn't a CFW being applied to LKE nodes in integration test runs. Attaching FW to LKE cluster upon creation is currently not available which is the reason why only Calico rules were only being applied. This PR fixes that problem by going through the test account after test execution and attaching a default firewall to all remaining instances

## ✔️ How to Test

Tested on my forked - https://github.com/ykim-1/terraform-provider-linode/actions/runs/11351444139/job/31572142776 

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

**Review by ezilber-akamai — APPROVED** [review]
Great addition. Nice work!

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -77,6 +77,60 @@ jobs:
...
+          mv calicoctl-linux-amd64 /usr/local/bin/calicoctl
+          mv kubectl /usr/local/bin/kubectl
+
+      - name: Apply Calico Rules to LKE # Only Running against Matrix USER 4 which includes LKE test suite
+        run: |
+          cd e2e_scripts/cloud_security_scripts/lke_calico_rules/ && ./lke_calico_rules_e2e.sh
+        env:
+          LINODE_TOKEN: ${{ secrets.LINODE_TOKEN_USER_4 }}
+
+  add-fw-lke-nodes:
```
Comment (lgarber-akamai) [inline .github/workflows/integration_tests.yml:106]: Would it make sense to add an `if: always()` here to also run this for failed integration test runs?

Comment (ykim-1) [reply .github/workflows/integration_tests.yml:106]: Yeah good catch

**Review by lgarber-akamai — APPROVED** [review]
Looks great, nice work!

**Pull request merged**